Material_Part4_Pitfalls/simple_math.py

```diff
@@ -0,0 +1,8 @@
+from math import radians, sin
+
+# Question: I was surprised that this threw an error in the mutable_default.py,
+# because apparently the import issue came up while importing datetime, which in turn imported math.
+# Does this mean generally that, if I happen to name a local module like a package imported by some of my imports, this will result in an error?
```

Comment:
Yes, exactly. Whenever a module tries to import something that is named the same as module in your package directory.